mongodb/client.bal

```diff
@@ -144,42 +109,32 @@ public isolated client class Client {
     remote isolated function insert(@display {label: "Document"} map<json> document, 
                            @display {label: "Collection Name"} string collectionName, 
                            @display {label: "Database Name"} string? databaseName = ()) returns Error? {
-        handle collection = check self.getCollection(collectionName, databaseName);
+        handle collection = check getCollection(self, collectionName, databaseName);
         string documentStr = document.toJsonString();
         return insert(collection, java:fromString(documentStr));
     }
 
     # Queries collection for documents, which sorts and limits the returned results.
     #
-    # + collectionName - Name of the collection
-    # + databaseName - Name of the database 
-    # + filter - Filter for the query
-    # + sort - Sort options for the query
-    # + limit - Limit options for the query results. No limit is applied for -1.
-    # + return - JSON array of the documents in the collection or else a `mongodb:Error` if unable to reach the DB
+    # + collectionName - Name of the collection  
+    # + databaseName - Name of the database  
+    # + filter - Filter for the query  
+    # + sort - Sort options for the query  
+    # + 'limit - The limit 
```

Comment:
Shall we describe this more clearly?

Reply:
Fixed : https://github.com/ballerina-platform/module-ballerinax-mongodb/pull/146/commits/88f012c01a4659c21d9754358f27d1f8a01553b6#diff-fc5ddf96bf9bd4eb5772667ec37d3ed50a3593b46b65688964828652ff96e3ecR32